webui/src/lib/components/repository/tree.jsx

```diff
@@ -72,6 +73,17 @@ const EntryRowActions = ({ repo, reference, entry, onDelete }) => {
         </Dropdown.Toggle>
 
         <Dropdown.Menu>
+          {entry.path_type === "object" && (
```

Comment:
This is a bit weird. Why not put all three items under the same conditional?

Reply:
because some have additional condition

Reply:
Not sure I understand the answer... unless I'm missing something...
You have three sibling components that are all conditionally rendered with `entry.path_type === "object"`, i.e., you're using the same condition three times.  
```suggestion
         {entry.path_type === "object" && (
    <>
        <Dropdown.Item
            onClick={(e) => {
            copyTextToClipboard(
                entry.physical_address
            );
            e.preventDefault();
            }}
        >
            <LinkIcon /> Copy Presigned URL
        </Dropdown.Item>
        <PathLink
        path={entry.path}
        reference={reference}
        repoId={repo.id}
        as={Dropdown.Item}
        presign={presign}
        >
            <DownloadIcon /> Download
        </PathLink>
        <Dropdown.Item
        onClick={(e) => {
            e.preventDefault();
            setShowObjectStat(true);
        }}
        >
        <InfoIcon /> Object Info
        </Dropdown.Item>
    </>
)}
```